radix-engine/src/system/actor.rs

```diff
@@ -2,12 +2,12 @@ use crate::internal_prelude::*;
 use crate::kernel::kernel_callback_api::CallFrameReferences;
 use radix_engine_interface::api::{AttachedModuleId, ModuleId};
 
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone, ScryptoSbor, PartialEq, Eq)]
```

Comment:
I think that this would be addressed by the `DebugTransactionReceipt` idea I mentioned [here](https://github.com/radixdlt/radixdlt-scrypto/pull/1834#issuecomment-2178658074). If we do it then we would no longer need to make those models SBOR encodable.

Reply:
So I was going to comment "we should remove Sbor from this and all these models"

But I guess this is so we can create the natural JSON output of the detailed trace for the tests?

I guess this is okay. We're happy to change these models because they are only used in tests :).

And we can catch it in future if someone tries to add this to an output model...